karspexet/venue/models.py

```diff
@@ -17,10 +18,16 @@ class SeatingGroup(models.Model):
     def __str__(self):
         return self.name
 
+    def active_pricing_model(self, timestamp=None):
+        return self.pricingmodel_set.active(timestamp).filter(seating_group_id=self.id).first()
```

Comment:
Actually, no. If `get` finds more than one object it will raise a `MultipleObjectsReturned` error. I will switch this back.